Formula/libpinyin.rb

```diff
@@ -0,0 +1,70 @@
+class Libpinyin < Formula
+  desc "Library to deal with pinyin"
+  homepage "https://github.com/libpinyin/libpinyin"
+  url "https://github.com/libpinyin/libpinyin/archive/2.6.0.tar.gz"
+  sha256 "2b52f617a99567a8ace478ee82ccc62d1761e3d1db2f1e05ba05b416708c35d2"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gnome-common" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "berkeley-db"
+  depends_on "glib"
+
+  resource "model" do
+    url "https://downloads.sourceforge.net/libpinyin/models/model19.text.tar.gz"
+    sha256 "56422a4ee5966c2c809dd065692590ee8def934e52edbbe249b8488daaa1f50b"
+  end
+
+  def install
+    resource("model").stage do |stage|
+      @modelpath = Dir.pwd
+      stage.staging.retain!
+    end
+    cp_r "#{@modelpath}/.", "./data/"
```

Comment:
I think this should work:
```suggestion
    resource("model").stage buildpath/"data"
```

Reply:
Thanks very much! It is much more elegant 👍

Reply:
Sure. Could you leave a comment before the `resource` block with a brief (max three lines) description of how to update the resource? It's not obvious if it follows a version scheme.

This will be very helpful if someone else tries to update this formula.

Reply:
Sure